tests/feature/factsphere/feature_test.go

```diff
@@ -0,0 +1,15 @@
+package factsphere
+
+import (
+	"testing"
+
+	"github.com/jakevoytko/crbot/testutil"
+)
+
+func TestKarma(t *testing.T) {
+	runner := testutil.NewRunner(t)
+
+	// Correct call format
+	runner.SendMessageWithoutResponse(testutil.MainChannelID, "?factsphere")
```

Comment:
Can we change this so that it asserts the number of messages that are in the test runner? Strictly speaking, this does have a response that's just not being asserted.